tests/rbd/rbd_features_disable.py

```diff
@@ -0,0 +1,97 @@
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def rbd_feature_disable(rbd, pool_type, **kw):
+    """
+    Disable the image features on image when image is moved to trash.
+    Args:
+        # feature_name: name of the feature to enable
+        rbd: rbd object
+        pool_type: pool type (ec_pool_config or rep_pool_config)
+        **kw: Test data
+    """
+    pool = kw["config"][pool_type]["pool"]
+    image = kw["config"][pool_type]["image"]
+    feature_name = "fast-diff"
+    try:
+        client = kw["ceph_cluster"].get_nodes(role="client")[0]
+        run_fio(image_name=image, pool_name=pool, client_node=client)
+
+        # Check if the feature is already enabled for the image
+        image_load = rbd.image_info(pool, image)
+        if feature_name in image_load.get("features", {}):
```

Comment:
remove this if-block if the RBD features are enabled by default. or add enable_feature in case if the feature is not enabled on the image.

Reply:
removed these lines since all images are enabled by default